mantle/cmd/kola/resources/iscsi_butane_setup.yaml

```diff
@@ -35,7 +39,10 @@ storage:
           inline: |
             #!/bin/bash
             set -xeuo pipefail
-            podman exec target bash -exc "
+            # Sleep for a second in case targetcli is not ready
+            # FIXME: use RestartMode=direct instead in the systemd unit but we need systemd v254
+            sleep 1
```

Comment:
I'm worried this could still flake on loaded CI systems. We could also do something like

```
while ! test -S /var/run/targetclid.sock; do sleep 1; done
```

inside the `podman exec` invocation.